### PR TITLE
build: add IC-2023.2.3 to the build matrix

### PR DESCRIPTION
**Pull request by angelozerr**
build: add IC-2023.2.3 to the build matrix

### CODE REVIEW

**Comment by fbricon** [comment]
2023.3 hasn't been released yet, you need to point to the EAP version

**Comment by angelozerr** [comment]
> 2023.3 hasn't been released yet, you need to point to the EAP version

Indeed I created this PR when teh EAP will be released.

File: .github/workflows/IJ.yml [hunk 1]
```diff
@@ -14,7 +14,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        IJ: [IC-2022.2, IC-2022.3, IC-2023.1, IC-2023.2, IC-2023.2.1]
```
Comment (fbricon) [inline .github/workflows/IJ.yml:17]: you should replace it with IC-2023.2.3 (released today) regardless of the 2023.3 EAP situation

Comment (angelozerr) [reply .github/workflows/IJ.yml:17]: thanks, fixed

**Pull request merged**